catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/RightsAttributes.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package ddf.catalog.data.impl.types;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.types.Rights;
+
+/**
+ * This class defines attributes that provide ownership restrictions on the metacard and/or resource.
+ */
+public class RightsAttributes implements Rights, MetacardType {
+
+    private static final Set<AttributeDescriptor> DESCRIPTORS;
+
+    private static final String NAME = "security";
+
+    static {
+        Set<AttributeDescriptor> descriptors = new HashSet<>();
+        descriptors.add(new AttributeDescriptorImpl(PRIVACY,
```

Comment:
I don't like the privacy attribute. It's too vague. There will be little or no consistency in when people set the flag or how the flag is interpreted. It will be junk data from day 1.

I'm confused why we need or want information about privacy, but if we have to have it, it makes more sense to do something like this (thanks Wikipedia):

Atrribute 1 PROTECTED - Protected information is not classified. Protected information pertains to any sensitive information that does not relate to national security and cannot be disclosed under the access and privacy legislation because of the possible injury to particular public or private interests.

And then use enumerations/CVE:

Protected C (Extremely Sensitive protected information): is used to protect extremely sensitive information, which if compromised, could reasonably be expected to cause extremely grave injury outside the national interest. Examples could include bankruptcy, identities of informants in criminal investigations, etc.

Protected B (Particularly Sensitive protected information): is used to protect information that could cause severe injury or damage to the people or group involved if it was released. Examples include medical records, annual personnel performance reviews, income tax returns, etc.

Protected A (Low-Sensitive protected information): is applied to low sensitivity information that should not be disclosed to the public without authorization and could reasonably be expected to cause injury or embarrassment outside the national interest. Example of Protected A information could include employee number, pay deposit banking information, etc.